documents/charter.md

```diff
@@ -4,7 +4,7 @@ While mobile network operators continuously invest in network resources, includi
 
 Video traffic usually employs adaptive bitrate (ABR) schemes to dynamically adjust the video quality (and thus the data rate) in response to changing network conditions. In the presence of traffic shaping, the ABR scheme should ideally adapt the quality and converge on a bitrate sustainable by the shaper. In practice this is extremely difficult to achieve while maintaining a good user experience. Application providers are even designing algorithms to detect the presence of such traffic shapers and estimate the targeted shaping rate, however, these algorithms are inaccurate and complex. Instead, it would be beneficial, for both the application provider and network operator, to signal the shaper rate to the application to self-adapt their video traffic to conform to the specified characteristics. Application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
 
-The Secure Communication of Network Properties (SCONE-PRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as maximum achievable bandwidth for a video. The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video using QUIC connections carrying those applications. Further use cases may be considered later in the working group, however, it is not assumpted that all signaling use cases must or can be addressed by by the same protocol.
+The Secure Communication of Network Properties (SCONE-PRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as the maximum achievable bandwidth for a video. The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video using QUIC connections carrying those applications. Further use cases may be considered later in the working group, however, it is not assumpted that all signaling use cases must or can be addressed by the same protocol. Any protocol specified by the working group should be based on a clearly defined use case. Design of a generic signaling protocol is explicitly out of scope for the working group.
```

Comment:
I think this is right. Since there's a PR, we should probably correct "assumpted" as "assumed".